ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -74,6 +74,17 @@
             }
         }
 
+        updateEnabled(enabled, state) {
+            if (this.widget) {
+                this.element.setAttribute(FormView.Constants.DATA_ATTRIBUTE_ENABLED, enabled);
```

Comment:
This should be included in FormFieldBase.js, as it's necessary for our runtime, such as setting the disable attribute and its related data attributes. Changes specific to HTML, like aria attributes, should belong to the view layer (i.e., datepickerview.js). You’ll also need to call super.updateEnabled for other functions, essentially invoking the method from FormFieldBase.